ecommerce/ecommerce/settings.py

```diff
@@ -105,7 +106,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'GMT'
```

Comment:
Hmm interesting, I've never actually changed this on an app before. Did you see something that suggested doing this or did it just seem like a good idea intuitively?

Reply:
Again, they said to change to your timezone in the tutorial so did it as well. Does it make any difference do you think?